core/domain/exp_jobs_one_off_test.py

```diff
@@ -414,6 +416,183 @@ def test_nonhuman_committers_not_counted(self):
             feconf.MIGRATION_BOT_USERNAME, exploration_summary.contributor_ids)
 
 
+class ExplorationContributorsSummaryOneOffJobTest(test_utils.GenericTestBase):
+    ONE_OFF_JOB_MANAGERS_FOR_TESTS = [
+        exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob]
+
+    EXP_ID = 'exp_id'
+
+    USERNAME_A = 'usernamea'
+    USERNAME_B = 'usernameb'
+    EMAIL_A = 'emaila@example.com'
+    EMAIL_B = 'emailb@example.com'
+
+    def setUp(self):
+        super(ExplorationContributorsSummaryOneOffJobTest, self).setUp()
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.signup(self.EMAIL_B, self.USERNAME_B)
+        self.process_and_flush_pending_tasks()
```

Comment:
Ok, so I tried your suggestion @maitbayev  and I have got 0 at all the places I tried to print self.count_jobs_in_taskqueue(). I have removed `self.count_jobs_in_taskqueue()` from `ExplorationContributorsSummaryOneOffJobTest` but I was not sure about the other two. Also, I tried removing all `self.process_and_flush_pending_task()` from the entire code and the tests still ran successfully. I guess we can safely remove it from the code given the situation. What do you think?